app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -26,21 +26,25 @@
             repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } ">
             <div>
               <span ng-bind-html="sclass.metadata.name  | highlight : $select.search"></span>
-              <span ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description)" class="text-muted">
-                <small>&ndash;
+              <div ng-if="sclass.parameters.type || sclass.parameters.zone || (sclass | description) || (sclass | storageClassAccessMode)" class="text-muted">
+                <small>
                   <span ng-if="sclass.parameters.type">
                     Type: {{sclass.parameters.type}}
                   </span>
                   <span ng-if="sclass.parameters.zone">
                     <span ng-if="sclass.parameters.type">|</span>
                     Zone: {{sclass.parameters.zone}}
                   </span>
-                  <span ng-if="sclass | description">
+                  <span ng-if="(sclass | storageClassAccessMode)">
                     <span ng-if="sclass.parameters.type || sclass.parameters.zone">|</span>
+                    Access: {{(sclass | storageClassAccessMode)}}
```

Comment:
nit: Parens aren't needed here. I think it's harder to read with them in this context since there are already curly braces.

```
Access: {{sclass | storageClassAccessMode}}
```